xml/System.Diagnostics/ProcessStartInfo.xml

```diff
@@ -1466,6 +1466,8 @@ p.WaitForExit();
 
 Setting this property to `false` enables you to redirect input, output, and error streams.
 
+This property has nothing to do with command shells (`bash`, `sh`) on Unix-like platforms. The term shell in the name here is a reference to the graphical shell on Windows. The word "shell" in this context means the same as "desktop", and lets users launch graphical applications or open documents.
```

Comment:
I'd probably rewrite the first two sentences a bit:

This property isn't related to command shells (`bash`, `sh`) on Unix-like platforms. The term shell in the API name is a reference to the graphical shell on Windows.